pandas/core/strings/object_array.py

```diff
@@ -382,6 +382,7 @@ def _str_get_dummies(self, sep: str = "|"):
 
         for i, t in enumerate(tags2):
             pat = sep + t + sep
+            # pylint: disable-next=cell-var-from-loop
             dummies[:, i] = lib.map_infer(arr.to_numpy(), lambda x: pat in x)
         return dummies, tags2
```

Comment:
is it possible to not turn off this check? e.g. before the for-loop, you could have some function like
```
        def _isin(test_elements, element):
            return element in test_elements
```
and then, inside the loop:
```
            dummies[:, i] = lib.map_infer(arr.to_numpy(), functools.partial(_isin, element=pat))
```

Reply:
Pylint agrees with you.